src/TaylorBot.Net/Program.Commands.Discord/src/TaylorBot.Net.Commands.Discord.Program/Modules/DiscordInfo/Commands/AvatarSlashCommand.cs

```diff
@@ -33,11 +40,17 @@ public class AvatarSlashCommand : ISlashCommand<AvatarSlashCommand.Options>
     {
         public SlashCommandInfo Info => new(AvatarCommand.Metadata.Name);
 
-        public record Options(ParsedUserOrAuthor user);
+        public record Options(ParsedUserOrAuthor user, ParsedOptionalString type);
```

Comment:
It's pretty convenient to use `string` but really we should parse this to an `enum` (like the last fm period for example), just so the type is more restrictive and we can catch errors in slash commands config